src/components/Permissions/UninstallButton.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import CircleButton from 'cozy-ui/transpiled/react/CircleButton'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import TrashIcon from 'cozy-ui/transpiled/react/Icons/Trash'
+import AppLinker, { generateWebLink } from 'cozy-ui/transpiled/react/AppLinker'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import { useClient } from 'cozy-client'
+
+export const UninstallButton = ({ matchingQueryResultData }) => {
+  const { t } = useI18n()
+  const client = useClient()
+  const { subdomain: subDomainType } = client.getInstanceOptions()
+  const appWebRef =
+    matchingQueryResultData &&
+    generateWebLink({
+      cozyUrl: client.getStackClient().uri,
+      slug: 'store',
+      subDomainType,
+      nativePath: `discover/${matchingQueryResultData.slug}/uninstall`
+    })
+  const appData = matchingQueryResultData && matchingQueryResultData
```

Comment:
`matchingQueryResultData && matchingQueryResultData` seems to be useless here. Or maybe I'm missing something?

What'is the difference with `const appData = matchingQueryResultData`?

However, if you apply my previous suggestion, then you would be able to remove this entire line